frontend/src/routes/Video.tsx

```diff
@@ -987,12 +992,21 @@ const DownloadButton: React.FC<{ event: SyncedEvent }> = ({ event }) => {
     );
 };
 
-const VideoShareButton: React.FC<{ event: SyncedEvent }> = ({ event }) => {
+type VideoShareButtonProps = {
+    event: Pick<SyncedEvent, "series" | "isLive" | "id" | "authorizedData">;
+    withDirectLink?: boolean;
```

Comment:
Maybe instead of this, make it `videoLink: string` and let the video page pass `window.location` and the manage page pass the direct link? Not super important though, if it weren't for that last comment, I wouldn't have remarked this :D